erpnext/selling/doctype/sales_order/test_sales_order.py

```diff
@@ -1217,6 +1217,19 @@ def test_so_optional_blanket_order(self):
 		# To test if the SO does NOT have a Blanket Order
 		self.assertEqual(so_doc.items[0].blanket_order, None)
 
+	def test_so_cancellation_when_si_drafted(self):
+		"""
+			Test to check if Sales Order gets submitted if Sales Invoice is in Draft state
+			Expected result: sales order should not get submitted 
```

Comment:
The test docstring, test name, PR name and PR description all seem to tell a different story. Please fix it.